lib/amorail/config.rb

```diff
@@ -5,7 +5,8 @@ class Config < Anyway::Config
 
     attr_config :usermail,
                 :api_key,
-                api_endpoint: 'https://new54c0b12948ffb.amocrm.ru',
+                :api_endpoint,
+                api_path: '/private/api/v2/json/',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.